ci/test.sh

```diff
@@ -29,5 +27,12 @@ fi
 
 env="$1"
 
-make storage
-tox -e flake8,test-$env,bench-$env
+if [ $env = "container" ]; then
+    make container
+    tox -e $env
```

Comment:
Can we do this inside tox instead?

For example:

    commands_pre = 
        container: make container

Reply:
We could if we add make to the allowlist_externals:
```
[testenv]
allowlist_externals = make
```

Reply:
I kept it in the `test.sh` script to be consistent with other tests with similar requirements (e.g., `make image` or `make storage`), which are not triggered from inside tox either.